src/main/app/src/app/admin/param-view/param-view.component.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+import {Component, OnInit} from '@angular/core';
+import {map} from 'rxjs/operators';
+import {Zaaktype} from '../../zaken/model/zaaktype';
+import {UtilService} from '../../core/service/util.service';
+import {ActivatedRoute} from '@angular/router';
+import {AutocompleteFormFieldBuilder} from '../../shared/material-form-builder/form-components/autocomplete/autocomplete-form-field-builder';
+import {ZaakafhandelParameters} from '../model/zaakafhandel-parameters';
+
+@Component({
+    templateUrl: './param-view.component.html',
+    styleUrls: ['./param-view.component.less']
+})
+export class ParamViewComponent implements OnInit {
```

Comment:
ParameterViewComponent of als je er in 1 keer meerdere parameters kunt bekijken ParametersViewComponent